_posts/2022-12-01-athena.md

```diff
@@ -0,0 +1,94 @@
+of Trino, without needing a team of experts to deploy and operate a Trino
+cluster for your organization. In fact, we wrote about this in the first edition
+of **Trino: The Definitive Guide**. There is also a section in the [new second
+edition]({% post_url 2022-10-03-the-definitive-guide-2 %}), that you can get for
+[free from Starburst](https://www.starburst.io/info/oreilly-trino-guide/).
+
+## Time flies
+
+Since the initial release however, time has not stood still. In fact, the Trino
+project has accelerated in [innovation, features, and releases tremendously]({%
+post_url 2022-08-04-decade-innovation %}). Amazon Athena users now get access to
+many of these great features. As [AWS mentions in the
+announcement](https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-athena-announces-upgraded-query-engine/),
+"over 50 new SQL functions, 30 new features, and more than 90 query performance
+improvements" are now available due the upgrade to a new version of Trino. These
+include [Row pattern recognition with MATCH_RECOGNIZE]({% post_url
+2021-05-19-row_pattern_matching %}), [new window features]({% post_url
+2021-03-10-introducing-new-window-features %}), support for ``UPDATE`` or
+``TRUNCATE`` statements, and many others.
+
+Performance improvements in our core engine as well as all the Trino connectors,
+show up in every release note. The [improvements observed by the Athena team in
+their
+benchmarks](https://aws.amazon.com/blogs/big-data/upgrade-to-athena-engine-version-3-to-increase-query-performance-and-access-more-analytics-features/)
+show the resulting gains nicely. This is great evidence that our approach of
+constantly working on small improvements, wherever we find potential, works
+well. This is necessary since we are already at a very high performance level,
+like an elite athlete, where every small improvement matters.
+
+Importantly also note that these improvements are in the Trino version of the
+engine only, since the [Presto project does not include these features]({%
+post_url 2022-08-02-leaving-facebook-meta-best-for-trino %}).
+
+## Client tools and collaboration
+
+Athena users also benefit from improvements for supporting client tools such as
+Python clients, dbt, Metabase and others. Working with other communities is of
+critical importance to the Trino project. The [innovations in our Iceberg
+connector]({{site.url}}/episodes/40.html), that are all now also available to
+Athena users, are a great example how we can lead the way together. Working with
+contributors from Amazon and other companies and projects has yielded some
+amazing improvements. In the [Trino summit and contributor
+congregation]({{site.url}}/episodes/42.html) we to reconnected in person and
+established even closer collaboration.
+
+## Looking forward
+
+So, what is next for Trino and Athena users? First up, you should upgrade to the
+new Trino engine in Athena, and avoid the legacy Presto engine.
```

Comment:
```suggestion
new Trino engine in Athena and avoid the legacy Presto engine.
```
This feels a little strongly-worded. I know we aren't fans of Presto, but there might be a more delicate way to put this while maintaining the same meaning.

Reply:
Nah ... lets call it out as it is ;-)

Reply:
We do still need to remove the comma here.